content/integrations/lambdatest-integration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: LambdaTest integration
+description: How to integrate your workflows with LambdaTest using codemagic.yaml
+weight: 6
+---
+
+**LambdaTest** is a cloud-based cross browser testing tool that allows you to perform cross browser testing for mobile applications. You can perform both manual and automated cross browser testing using LambdaTest. More information can be found here. 
+
+## Create a LambdaTest account
+
+Registering with LambdaTest is required in order to be able to get the username and access token. You can sign up for free [here](https://www.lambdatest.com/).
+
+## Configuring Real Time and App Automation with Codemagic
+
+**LambdaTest** offers two testing environments, **Real Time** and **App Automation**. Applications can be submitted to both testing environments through Codemagic using a cURL request. In order to configure them correctly, you will need two pieces of data: **username** and **access token**. They can be found in the **LambdaTest UI** with your account. 
+
+## Real Time and App Automation
+
+You can test your **.ipa** and **.apk** directly on real devices by submitting them to the **LambdaTest** environment via a cURL request:
+
+```
+-Scripts:
+    - name: Submitting app to LambdaTest:
+    script: |
+        curl --location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' --header 'Authorization: Basic $LAMBDATEST' --form 'name="lambda1"' --form 'appFile=@"app/build/outputs/apk/release/app-release.apk"'
+```
+
+**$LAMBDATEST** environment variable is a custom name and can be named to anything preferred. More info can be found [here](https://docs.codemagic.io/variables/environment-variable-groups/) about how to set up variable groups with Codemagic. The environment variable is the base64 encoded form of your username and access token. They need to be encoded with the following format [here](https://mixedanalytics.com/knowledge-base/api-connector-encode-credentials-to-base-64/):
```

Comment:
Perhaps just change the order of information here. After you say which information is needed to integrate Lambdatest and Codemagic, I would continue with encoding the username and token and saving the encoded string to Codemagic as an environment variable. So the steps would be as follow:
1) Encode and save the username and access token into an env var
2) Run the scripts in yaml referencing that env var

Reply:
Changes were made.